src/components/templates/Video/HomeVideoList/index.tsx

```diff
@@ -1,46 +1,80 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { getVideoListPopular, getVideoListComplete } from "apis/video";
 import { VideoCard } from "components/modules";
+import { useQuery } from "@tanstack/react-query";
 import * as S from "./style";
 
 const videolist = [
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 1,
+    title: "뉴진스 민지",
+    link: "https://youtu.be/p1cE9T0CFCQ",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 2,
+    title: "엔시티 런쥔",
+    link: "https://youtu.be/mhJfozWlSaM",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 3,
+    title: "소녀시대 윤아",
+    link: "https://youtu.be/uoxcux4Scxc",
   },
   {
-    title: "title",
-    video: "video",
-    like: 10,
+    videoIdx: 4,
+    title: "뉴진스 하니",
+    link: "https://youtu.be/lmJPeFW75qQ",
   },
 ];
 
 function HomeVideoList() {
+  const populerQuery = useQuery(["popularVideo"], getVideoListPopular);
+  const [popularVideo, setPopularVideo] = useState([]);
+
+  const newQuery = useQuery(["newVideo"], getVideoListComplete);
+  const [newVideo, setNewVideo] = useState([]);
+
+  useEffect(() => {
+    if (populerQuery.data) {
+      setPopularVideo(populerQuery.data);
+    }
+  }, [populerQuery.data]);
+
+  useEffect(() => {
+    if (newQuery.data) {
+      setNewVideo(newQuery.data);
+    }
+  }, [newQuery.data]);
+
   return (
     <S.VideoListContainer>
       <S.ListTitle>인기 동영상</S.ListTitle>
       <S.VideoListBlock>
-        {videolist &&
-          videolist.map((v: { title: string; video: string; like: number }) => (
-            <VideoCard title={v.title} video={v.video} like={v.like} />
-          ))}
+        {popularVideo &&
+          popularVideo.map(
```

Comment:
만약에 useQuery의 data만 사용하게되면,
처음에는 data가 undefined 라서 map 함수를 사용할 수 없다는 오류가 발생해요.

그래서 저는 보통 if (isLoading) return '로딩중입니다' 같은 방식으로 사용합니다.
```suggestion
if (isLoading) return <div>"로딩중"</div>

return (<S.vieoListContainer>
    ....
    )
```
react-quert가 자동으로 loading, success, error를 관리하고 갱신해줘서, 별다른 갱신 로직은 없어도 될거같아요!